spec/basic/expected_conditions_spec.js

```diff
@@ -66,6 +66,18 @@ describe('expected conditions', function() {
     expect(EC.titleIs('My AngularJS App').call()).toBe(true);
   });
 
+  it('should have urlContains', function() {
+      var baseUrlFromSpec = browser.baseUrl;
+      expect(EC.urlContains('/form').call()).toBe(true);
+      expect(EC.urlContains(baseUrlFromSpec+ 'index.html#/form').call()).toBe(true);
+  });
+
+  it('should have urlIs', function() {
+      var baseUrlFromSpec = browser.baseUrl;
+      expect(EC.urlIs('http://localhost:8081').call()).toBe(false);
```

Comment:
@juliemr  @cnishina 
`baseUrl: env.baseUrl + '/ng1/',` is what I am seeing in the conf file. 
So its the same I believe.
Refer: https://github.com/angular/protractor/blob/master/spec/basicConf.js#L21